Jellyfin.Windows.Tray/TrayApplicationContext.cs

```diff
@@ -122,7 +125,16 @@ private void LoadPortFromJellyfinConfig()
                     FirstRunDone = SettingsReader.SelectSingleNode("/ServerConfiguration/IsStartupWizardCompleted").ValueAsBoolean;
                     string port = SettingsReader.SelectSingleNode("/ServerConfiguration/PublicPort").Value;
 
-                    _localJellyfinUrl = "http://localhost:" + port + "/web/index.html";
+                    XDocument networkXml = XDocument.Load(_networkFile);
+                    XPathNavigator NetworkReader = networkXml.CreateNavigator();
+
+                    NetworkAddress = NetworkReader.SelectSingleNode("/NetworkConfiguration/LocalNetworkAddresses").Value;
+
+                    if (string.IsNullOrEmpty(NetworkAddress)) {
+                        _localJellyfinUrl = "http://localhost:" + port + "/web/index.html";
```

Comment:
If you, instead, set the `NetworkAddress` to `localhost` here you can prevent duplicating this string concatenation.